Deployments/demov3/cisco/demo-core-cisco-1-firewall-nsg/diagram/getPUMLFromJSONParm.ps1

```diff
@@ -1,3 +1,4 @@
+# Ex: .\cleantemplate.ps1 ..\parameters\masterdeploy.parameters.json -outFileName dependsOn.puml
```

Comment:
Good to add comments to help, but you might want to look at the format that supports the built-in `Get-Help` commandlet https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_comment_based_help?view=powershell-6